buildenv/jenkins/common/build.groovy

```diff
@@ -814,12 +814,40 @@ def _build_all() {
     }
 }
 
+// TODO: remove this workaround when https://github.com/adoptium/infrastructure/issues/3597 resolved. related: infra 9292
+def create_docker_image_locally()
+{
+    new_image_name=DOCKER_IMAGE.split(':')[0]+'_cuda'
+    sh '''
+        echo 'ARG image
+            ARG cuda_ver=12.2.0
+            ARG cuda_distro=ubi8
+            FROM nvidia/cuda:${cuda_ver}-devel-${cuda_distro} as cuda
+            FROM $image
+            RUN mkdir -p /usr/local/cuda/nvvm
+            COPY --from=cuda /usr/local/cuda/include         /usr/local/cuda/include
+            COPY --from=cuda /usr/local/cuda/nvvm/include    /usr/local/cuda/nvvm/include
+            ENV CUDA_HOME="/usr/local/cuda"' > dockerFile
+    '''
+    println "Preparing Docker image ${new_image_name} locally ..."
+    dockerRegistry = getDockerRegistry(DOCKER_IMAGE)
+    dockerCredentialID = variableFile.get_user_credentials_id(dockerRegistry.replaceAll('https://','') ?: 'dockerhub')
+    docker.withRegistry(dockerRegistry, "${dockerCredentialID}") {
+            docker.build(new_image_name, "--build-arg image=${DOCKER_IMAGE} -f dockerFile .")
+    }
+    DOCKER_IMAGE = new_image_name
+}
+
 def build_all() {
     stage ('Queue') {
         timeout(time: 10, unit: 'HOURS') {
             node("${NODE}") {
                 timeout(time: 5, unit: 'HOURS') {
                     if ("${DOCKER_IMAGE}") {
+                        // TODO: remove this workaround when https://github.com/adoptium/infrastructure/issues/3597 resolved. related: infra 9292
+                        if ((SDK_VERSION.toInteger() >= 17 && PLATFORM ==~ /x86-64_linux.*/) || (PLATFORM ==~ /ppc64le_linux.*/ )) {
```

Comment:
This broken the acceptance build for the head stream because "next" is not an integer.
See https://openj9-jenkins.osuosl.org/job/Build_JDKnext_aarch64_linux_OpenJDK/586 for example.

Reply:
See #19868

Reply:
#19868 is closed; but https://github.com/eclipse-openj9/openj9/pull/19869 should fix this.

Reply:
https://github.com/eclipse-openj9/openj9/pull/19870